config/production.yaml.example

```diff
@@ -100,6 +100,7 @@ log:
     maxFileSize: 12MB
     maxFiles: 20
   anonymizeIP: false
+  log_ping_requests: false
```

Comment:
Let this to `true`, and if the admin uses it for health check it can disable it

Reply:
Fixed